specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -1920,6 +2240,339 @@
           "description": "A textual explanation that can be used for display purposes to summarize the index value and category. For example, when the index value for Flight Delays is very good, the description will be \"Conditions are excellent for flying!\"."
         }
       }
+    },
+    "ActiveStormResponse": {
+      "type": "object",
+      "description": "all government-issued active storms",
+      "properties": {
+        "results": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ActiveStorm"
+          },
+          "description": "all government-issued active storms"
+        }
+      }
+    },
+    "ActiveStorm": {
+      "type": "object",
+      "description": "Government-issued active storm",
+      "properties": {
+        "year": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Year of origination"
+        },
+        "basinId": {
+          "$ref": "#/definitions/basinID",
+          "description": "Basin identifier (AL, EP, SI, NI, CP, NP, SP)"
+        },
+        "name": {
+          "type": "string",
+          "description": "The name of the depression."
+        },
+        "isActive": {
+          "type": "boolean",
+          "description": "True if the depression has been updated recently."
+        },
+        "isSubtropical": {
+          "type": "boolean",
+          "description": "True when the depression is classified as a subtropical cyclone."
+        },
+        "govId": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Government storm ID. This will match the depression number."
+        }
+      }
+    },
+    "basinID": {
```

Comment:
definition name could follow existing way to be pascal case.

Reply:
Updated